pkg/files/file_helpers_benchmark_test.go

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) F5, Inc.
+//
+// This source code is licensed under the Apache License, Version 2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+
+// Package files implements utility routines for gathering information about files and their contents.
+package files
+
+import (
+	"testing"
+
+	mpi "github.com/nginx/agent/v3/api/grpc/mpi/v1"
+)
+
+func BenchmarkGenerateHash(b *testing.B) {
```

Comment:
Need to update the makefile `performance-test` and `generate-pgo-profile` targets to run this benchmark

Reply:
Will I leave this until we figure out if we are changing the way the hash is done ? since we are adding more tests ?